starter-code/css/main.css

```diff
@@ -7,4 +7,165 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+height: 100%;
+  width: 100%;
 }
+
+
+.css3gradient{width:1431px;height:1966px;
+ background-color:##06365f;
```

Comment:
This line brings up an error in the Inspector tool because you've included two hashtags. When working with colors, you should only include one.